src/main/java/com/mraof/minestuck/block/machine/ComputerBlock.java

```diff
@@ -161,18 +238,37 @@ private void dropItems(Level level, int x, int y, int z, BlockState state)
 			float rx = rand.nextFloat() * 0.8F + 0.1F;
 			float ry = rand.nextFloat() * 0.8F + 0.1F;
 			float rz = rand.nextFloat() * 0.8F + 0.1F;
-			ItemEntity entityItem = new ItemEntity(level, x + rx, y + ry, z + rz, ProgramData.getItem(program));
+			ItemStack diskStack = ProgramData.getItem(program);
+			if(diskStack != null && !diskStack.isEmpty())
+			{
+				ItemEntity entityItem = new ItemEntity(level, x + rx, y + ry, z + rz, diskStack);
+				entityItem.setDeltaMovement(rand.nextGaussian() * factor, rand.nextGaussian() * factor + 0.2F, rand.nextGaussian() * factor);
+				level.addFreshEntity(entityItem);
+			}
+		}
+		
+		for(int iterate = 0; iterate < be.blankDisksStored; iterate++)
+		{
+			float rx = rand.nextFloat() * 0.8F + 0.1F;
+			float ry = rand.nextFloat() * 0.8F + 0.1F;
+			float rz = rand.nextFloat() * 0.8F + 0.1F;
+			ItemEntity entityItem = new ItemEntity(level, x + rx, y + ry, z + rz, MSItems.BLANK_DISK.get().getDefaultInstance());
 			entityItem.setDeltaMovement(rand.nextGaussian() * factor, rand.nextGaussian() * factor + 0.2F, rand.nextGaussian() * factor);
 			level.addFreshEntity(entityItem);
 		}
+		
 		if(state.getValue(STATE) == State.BROKEN)
 		{
 			float rx = rand.nextFloat() * 0.8F + 0.1F;
 			float ry = rand.nextFloat() * 0.8F + 0.1F;
 			float rz = rand.nextFloat() * 0.8F + 0.1F;
-			ItemEntity entityItem = new ItemEntity(level, x + rx, y + ry, z + rz, ProgramData.getItem(-1));
-			entityItem.setDeltaMovement(rand.nextGaussian() * factor, rand.nextGaussian() * factor + 0.2F, rand.nextGaussian() * factor);
-			level.addFreshEntity(entityItem);
+			ItemStack diskStack = ProgramData.getItem(-1);
+			if(diskStack != null)
```

Comment:
`ProgramData.getItem()` shouldn't be returning null, so this check should not be needed.